Assets/Scripts/Matchplay/Client/Services/MatchplayMatchmaker.cs

```diff
@@ -34,6 +34,19 @@ public class MatchplayMatchmaker : IDisposable
         CancellationTokenSource m_CancelToken;
         const int k_GetTicketCooldown = 1000;
 
+        public MatchplayMatchmaker()
+        {
+#if AUTHENTICATION_TESTING_STAGING_UAS
+            SetStagingEnvironment();
+#endif
+        }
+
+        private async void SetStagingEnvironment()
+        {
+            await AuthenticationWrapper.Authenticating();
+            var sdkConfiguration = (IMatchmakerSdkConfiguration)MatchmakerService.Instance;
+            sdkConfiguration.SetBasePath("https://matchmaker-stg.services.api.unity.com");

```

Comment:
Is it alright to expose this url to the public?

Reply:
Not a security concern but might not be the cleanest to put in a sample project when It could be directly in the sdk code. I'll check with my lead, will let you know :)